styles/base.css

```diff
@@ -204,22 +176,15 @@ h6 {
 /* Header */
 
 :root {
-    --header-margin-left: -5px;
     --logo-lg-width: 15rem;
 }
 
 @media (min-width: 992px) {
     :root {
-        --header-margin-left: -10px;
         --logo-lg-width: 20rem;
     }
 }
 
-header {
-    margin-left: var(--header-margin-left);
-    /* account for padding within the Compiler logo*/
-}
-
```

Comment:
This is no longer necessary, b/c the logo file is updated to be tightly cropped.